source/main.c

```diff
@@ -319,15 +320,22 @@ static int LoadSounds(void* data)
 
 void update_usb_path(char* path)
 {
-	for (int i = 0; i <= MAX_USB_DEVICES; i++)
+	if(apollo_config.hddpath)
 	{
-		sprintf(path, USB_PATH "PS4/", i);
-
+		sprintf(path, APOLLO_PATH "PS4/");
 		if (dir_exists(path) == SUCCESS)
 			return;
 	}
-
```

Comment:
Just keep the original search here, then when out of the loop, the code can check
```c
if (dir_exists("/data/fakeusb/PS4/") == SUCCESS)
    sprintf(path, "/data/fakeusb/PS4/");
```

Reply:
Fixed